drivers/wireless/gs2200m.c

```diff
@@ -85,6 +85,7 @@
 #define NRESPMSG     (16 + 2)
 
 #define MAX_PKT_LEN  1500
+#define MAX_PAYLOAD  (MAX_PKT_LEN - 36)
```

Comment:
You should not hardcode the number 36.

What is this numberanyway.  It should:

The size of the Ethernet header (14) +
The size of IPv4 or IPv6 header (20 or 40) +
Any other headers.

See:
ethernet.h:#define ETH_HDRLEN       14     /* Header size: 2*6 + 2 */
ip.h:#  define IPv4_HDRLEN     20    /* Size of IPv4 header (without options) */
ip.h:#  define IPv6_HDRLEN     40    /* Size of IPv6 header */

36 does not see right?  Is ther ea 2 byte FCS?  Shouldn't this handle IPv6 too?

Let's not use hard-coded values.

Reply:
Hello @patacongo 

Thanks fot the comments.
Actually, It it not an IP header nor TCP header.
It's just a GS2200M command header + guard size.

Anyway I'll define the value later.

Reply:
>Anyway I'll define the value later.

Hi @patacongo,

I've just pushed the code again.